core/common/src/-CommonPlatform.kt

```diff
@@ -29,6 +29,7 @@ internal expect fun String.asUtf8ToByteArray(): ByteArray
 public expect open class IOException(message: String?, cause: Throwable?) : Exception {
     public constructor()
     public constructor(message: String?)
+    public constructor(cause: Throwable?)
```

Comment:
It does make sense to also add a constructor accepting both a message and a cause.

Reply:
It's already handled by the primary constructor.